liblangutil/SemVerHandler.h

```diff
@@ -25,17 +25,27 @@
 
 #include <liblangutil/Token.h>
 #include <libsolutil/Assertions.h>
+#include <liblangutil/Exceptions.h>
 
 #include <string>
 #include <optional>
 #include <utility>
 #include <vector>
+#include <variant>
 
 namespace solidity::langutil
 {
 
-class SemVerError: public util::Exception
+struct SemVerError: public Error
```

Comment:
I think it would be easier if this was not part of the Solidity error inheritance hierarchy. You don't make use of most of the features (source location etc) and you would not need to assign a new error id for each of the tiny ways the parser can fail.

Reply:
This does not seem resolved yet.